### DISCUSSION TITLE
How do I search output from TSInstallInfo?

### DISCUSSION THREAD

**Discussion by smartding** [nvim-treesitter/nvim-treesitter#7485] [Q&A] [answered]
TSInstallInfo outputs to a temporary window where I can't search.  The window looks as follows:
Is there any configuration I can use to let TSInstallInfo output somewhere else, where I can search?  It's a long list...
![image](https://github.com/user-attachments/assets/466a757e-7bc2-432b-895f-68904bb5022d)

**Comment by clason** [comment, marked as answer]
This is the standard `:messages` output. You can paste it into an open buffer with `:pu=execute('TSInstallInfo')`.
